plugins/custom_experiments/autoconf/autoconf/AutoGluonModels/README.md

```diff
@@ -0,0 +1,37 @@
+# min_gpu_recommender Model Information
```

Comment:
Please run Markdownlint on this file to fix the warnings. I'll have to look into why the CI doesn't pick this up

```
markdownlint-cli2 "**/*.md"
```

Reply:
Ran the markdown through `prettier` and fixed the overflows